tmt/steps/report/polarion.py

```diff
@@ -35,77 +36,106 @@ class ReportPolarionData(tmt.steps.report.ReportStepData):
+        is_flag=True,
+        help='Use hostname and arch from guest facts.'
         )
 
     planned_in: Optional[str] = field(
         default=None,
         option='--planned-in',
         metavar='PLANNEDIN',
-        help='Select a specific release to mark this test run with'
+        help=(
+            'Select a specific release to mark this test run with, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_PLANNED_IN.')
         )
 
     assignee: Optional[str] = field(
         default=None,
         option='--assignee',
         metavar='ASSIGNEE',
-        help='Who is responsible for this test run'
+        help=(
+            'Who is responsible for this test run, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_ASSIGNEE.')
         )
 
     pool_team: Optional[str] = field(
         default=None,
         option='--pool-team',
         metavar='POOLTEAM',
-        help='Which subsystem is this test run relevant for'
+        help=(
+            'Which subsystem is this test run relevant for, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_POOL_TEAM')
         )
 
     arch: Optional[str] = field(
         default=None,
         option='--arch',
         metavar='ARCH',
-        help='Which architecture was this run executed on'
+        help=(
+            'Which architecture was this run executed on, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_ARCH.')
         )
 
     platform: Optional[str] = field(
         default=None,
         option='--platform',
         metavar='PLATFORM',
-        help='Which platform was this run executed on'
+        help=(
+            'Which platform was this run executed on, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_PLATFORM.')
         )
 
     build: Optional[str] = field(
         default=None,
         option='--build',
         metavar='BUILD',
-        help='Which build was this run executed on'
+        help=(
+            'Which build was this run executed on, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_BUILD.')
         )
 
     sample_image: Optional[str] = field(
         default=None,
         option='--sample-image',
         metavar='SAMPLEIMAGE',
-        help='Which sample image was this run executed on'
+        help=(
+            'Which sample image was this run executed on, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_SAMPLE_IMAGE.')
         )
 
     logs: Optional[str] = field(
         default=None,
         option='--logs',
         metavar='LOGLOCATION',
-        help='Location of the logs for this test run'
+        help=(
+            'Location of the logs for this test run, '
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_LOGS.')
         )
 
     composeid: Optional[str] = field(
         default=None,
         option='--composeid',
```

Comment:
```suggestion
        option='--compose-id',
```

Is it too late to make the option consistent with others and use dash to separate words?

Reply:
yeah, the problem is on polarion side, it's one word there and I don't want to do some magic splitting/merging on tmt side for one variable